packages/contracts-bedrock/src/L1/SuperchainConfig.sol

```diff
@@ -190,4 +178,10 @@ contract SuperchainConfig is Initializable, ISemver {
     function dependencySet() external view returns (uint256[] memory) {
         return _dependencySet.values();
     }
+
+    /// @notice Returns the size of the dependency set.
+    /// @return The size of the dependency set.
+    function dependencySetSize() external view returns (uint8) {
```

Comment:
Do we really need this function?

Reply:
I think it can be useful for off-chain tooling

Reply:
But can't you easily do `depdendencySet().length`?

Reply:
I mean, u can, but this way is simpler than decoding the array and also less data to return

Reply:
If u need to access from another contract is O(1) vs O(n)

Reply:
Ok I'm sold, thanks for the answer!